docs/mutations.rst

```diff
@@ -33,18 +33,21 @@ In this example we have the following elements:
 
 ``Mutation`` type with two mutations: ``login`` mutation that requires username and password strings and returns bool with status, and ``logout`` that takes no arguments and just returns status.
 
-For the sake of simplicity, our mutations return bools, but really there is no such restriction. You can have a resolver that returns status code, an updated object, or an error message::
+For the sake of simplicity, our mutations return bools, but really there is no such restriction. In fact, it is recommended for mutations to return dedicated ``Payload`` types:
```

Comment:
Missing double colon (`::`) at the end of paragraph.

Also, lets replace "there is no such restriction" with "there is no such requirement", and second part to `it is recommended as good practice that mutations return dedicated "payload" types::`